src/styles/generator.scss

```diff
@@ -211,3 +211,26 @@ h2 {
     margin-top: 10px;
     margin-bottom: 10px;
 }
+
+/*Displaying wizard text based on screen size*/
+.p-wizard-desktop {
+    display: none;
+}
+
+@media screen and (max-width: 980px) {
```

Comment:
Initially, I wanted to suggest using our [`mobile` mixin](https://github.com/datenanfragen/website/blob/master/src/styles/variables.scss#L47).

But then I noticed (and remembered…) that we don't even need that. We already have classes `.mobile-only` and `.desktop-only` ([here](https://github.com/datenanfragen/website/blob/master/src/styles/framework.scss#L107)) that do the exact same thing.

Let's just use those and simplify the PR.

Reply:
Okay, thanks for your feedback. I'll fix that asap.